### ISSUE TITLE
Bug: DAO heirarchy cards are too dark

### ISSUE THREAD

**Issue by herbig** [decent-dao/fractal-interface#1071]
See screenshot compared to the rest of the app

<img width="471" alt="Screenshot 2023-03-17 at 10 31 34 PM" src="https://user-images.githubusercontent.com/384559/226078539-a8a213fa-ee5b-49b3-b8fa-40b066811004.png">

<img width="438" alt="Screenshot 2023-03-17 at 10 31 43 PM" src="https://user-images.githubusercontent.com/384559/226078544-6a66f394-9f13-4a3a-8134-aea9cbd5b1c0.png">

**Comment by Da-Colon** [comment]
yeah I mentioned I believe in the PR. but the darker color is due to the lines being hidden behind as the proper BG is `black.900-semi-transparent`. I'm thinking the fix lies with some of the work you had started @herbig with using refs to identify where the node cards bottoms are.

![image](https://user-images.githubusercontent.com/38386583/226135798-b40cd79f-f8e5-4f2f-8fbb-0379360b81d0.png)
